pkg/validator/cluster.go

```diff
@@ -539,6 +539,11 @@ func validateRoleServiceAccount(
 				"service account "+role.ServiceAccountName+" requested by role "+role.Name+" does not exist")
 			continue
 		}
+		// Convert k8s.io/api/authentication/v1".ExtraValue -> k8s.io/api/authorization/v1".ExtraValue
+		var xtra map[string]sar.ExtraValue
```

Comment:
I think to avoid assignment to a nil map this would need to be:
```golang
xtra := make(map[string]sar.ExtraValue)
```

(Unfortunately I don't know off the top of my head how to actually stimulate the API server into putting values in this field for testing.)